pgml-extension/Cargo.toml

```diff
@@ -7,13 +7,14 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [features]
-default = ["pg13"]
```

Comment:
Since this is actually python (including huggingface)? I'd rename this feature to `python`

Reply:
Yeah true that.